src/main/java/utils/InverseFloatingArray.java

```diff
@@ -0,0 +1,39 @@
+package utils;
+
+import java.io.IOException;
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.List;
+
+public class InverseFloatingArray {
+
+    private List<String> outputMessages = new ArrayList<>();
+
+    private String className = InverseFloatingArray.class.getSimpleName();
+
+    public void readInput() throws IOException {
+        List<String> input = FileUtils.readFromTxt(className);
+
+        input.forEach(this::inverseArray);
+
+        FileUtils.writeToTxt(className, outputMessages);
+    }
+
+    public void inverseArray(String input){
+        String[] inputArray = input.split(" ", -1);
+        float[] floatArray = new float[inputArray.length];
+
+        String message = "";
+
+        for(int i = 0, j = 1; i < inputArray.length; i++, j++){
```

Comment:
remove j variable
use inputArray.length - (i + 1) instead of using j variable to be incremented

Also you can use a  StringBuilder to add the output result, to avoid creation of multiple objects of String in the memory